plugins/hls-cabal-plugin/test/CabalAdd.hs

```diff
@@ -122,6 +124,23 @@ cabalAddTests =
                                    [ ("3d-graphics-examples", T.empty)
                                    , ("3d-graphics-examples", "1.1.6")
                                    ]
+    , testHiddenPackageSuggestions "Check CabalAdd's parser, with version, with PackageImports"
+                                   [ "(needs flag -package-id base-0.1.0.0)"
+                                   , "(needs flag -package-id Blammo-wai-0.11.0)"
+                                   , "(needs flag -package-id BlastHTTP-2.6.4.3)"
+                                   , "(needs flag -package-id CC-delcont-ref-tf-0.0.0.2)"
+                                   , "(needs flag -package-id 3d-graphics-examples-1.1.6)"
+                                   , "(needs flag -package-id AAI-0.1)"
+                                   , "(needs flag -package-id AWin32Console-1.19.1)"
+                                   ]
+                                   [ ("base","0.1.0.0")
+                                   , ("Blammo-wai", "0.11.0")
+                                   , ("BlastHTTP", "2.6.4.3")
+                                   , ("CC-delcont-ref-tf", "0.0.0.2")
+                                   , ("3d-graphics-examples", "1.1.6")
+                                   , ("AAI", "0.1")
+                                   , ("AWin32Console", "1.19.1")
+                                   ]
```

Comment:
I did not add the "without version" test because I am not sure if the "without version" case exists when `PackageImports` is used.